PWGCF/MultiparticleCorrelations/Core/MuPa-MemberFunctions.h

```diff
@@ -1381,36 +1812,43 @@ void StoreLabelsInPlaceholder()
   } // now it's safe :-)
 
   // b) Fetch TObjArray with labels from an external file:
-  TObjArray* oa = GetObjArrayWithLabels(fFileWithLabels.Data());
+  TObjArray *oa = GetObjArrayWithLabels(fFileWithLabels.Data());
   if (!oa) {
-    cout << __LINE__ << endl;
-    cout << Form(" fFileWithLabels = %s", fFileWithLabels.Data()) << endl;
-    exit(1);
+    LOGF(info, "\033[1;33m fFileWithLabels = %s \033[0m",
+         fFileWithLabels.Data());
+    LOGF(fatal, "in function \033[1;31m%s at line %d\033[0m",
+         __PRETTY_FUNCTION__, __LINE__);
   }
 
   // c) Book the placeholder fTest0LabelsPlaceholder for all labels:
   Int_t nLabels = oa->GetEntries();
-  fTest0LabelsPlaceholder = new TH1I("fTest0LabelsPlaceholder", Form("placeholder for all labels, %d in total", nLabels), nLabels, 0, nLabels);
+  fTest0LabelsPlaceholder =
+      new TH1I("fTest0LabelsPlaceholder",
+               Form("placeholder for all labels, %d in total", nLabels),
+               nLabels, 0, nLabels);
   fTest0LabelsPlaceholder->SetStats(kFALSE);
 
   // d) Finally, store the labels from external source into placeholder:
   Int_t bin = 1; // used only for fTest0LabelsPlaceholder
   Int_t order = -44;
   for (Int_t e = 0; e < nLabels; e++) {
-    TObjArray* temp = TString(oa->At(e)->GetName()).Tokenize(" ");
+    TObjArray *temp = TString(oa->At(e)->GetName()).Tokenize(" ");
     if (!temp) {
-      cout << __LINE__ << endl;
-      exit(1);
+      LOGF(fatal, "in function \033[1;31m%s at line %d\033[0m",
+           __PRETTY_FUNCTION__, __LINE__);
     }
     order = temp->GetEntries();
     delete temp; // yes, otherwise it's a memory leak
     if (0 == order) {
       continue;
     } // empty lines, or the label format which is not supported
     // 1-p => 0, 2-p => 1, etc.:
-    t0_a.fTest0Labels[order - 1][counter[order - 1]] = new TString(oa->At(e)->GetName()); // okay...
-    fTest0LabelsPlaceholder->GetXaxis()->SetBinLabel(bin++, t0_a.fTest0Labels[order - 1][counter[order - 1]]->Data());
-    // cout<<__LINE__<<": "<<t0_a.fTest0Labels[order-1][counter[order-1]]->Data()<<endl;
+    t0_a.fTest0Labels[order - 1][counter[order - 1]] =
+        new TString(oa->At(e)->GetName()); // okay...
+    fTest0LabelsPlaceholder->GetXaxis()->SetBinLabel(
+        bin++, t0_a.fTest0Labels[order - 1][counter[order - 1]]->Data());
+    // cout<<__LINE__<<":
+    // "<<t0_a.fTest0Labels[order-1][counter[order-1]]->Data()<<endl;
     counter[order - 1]++;
     // cout<<TString(line).Data()<<endl;
     // cout<<oa->GetEntries()<<endl;
```

Comment:
Please, don't leave code commented

Reply:
I might still need this line during local debugging, but it will be removed eventually.